.changeset/rich-dragons-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-core": minor
+---
+
+useOnMountEffect hook
```

Comment:
**nit:** I find it useful for our release notes to include a verb describing what happened, like "Added  " or "Fixed ".